src/components/Viewer/Panel/Qualification.jsx

```diff
@@ -12,46 +13,81 @@ import en from '../locales/fr.json'
 
 const locales = { fr, en }
 
-const Qualification = ({ file, t }) => {
-  const { name, metadata } = file || {}
-  const qualificationLabel = metadata?.qualification.label
+const validPageLabel = page => page === 'front' || page === 'back'
+
+const Qualification = ({ file, t, f, lang }) => {
```

Comment:
Instead of 
```
l.23  const { name: filename, metadata } = file || {}
```
 If file is undefined, so we can give it a default value 
```
const Qualification = ({ file = {}, t, f, lang }) => {
```
Same thing about metadata and qualification objects.
What do you think ?

Reply:
En effet, c'est une autre approche, peut-être un peu plus lisible 👍